airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -98,3 +99,24 @@ def get_job_group(self, job_id: int, embed: str, include_deleted: bool) -> Dict[
         response = requests.get(url, headers=self._headers, params=params)
         response.raise_for_status()
         return response.json()
+
+    @retry(stop=stop_after_attempt(5), wait=wait_exponential(multiplier=1, max=10))
+    def run_job_group(self, recipe_id: int) -> Dict[str, Any]:
+        """
+        Creates a jobGroup, which launches the specified job as the authenticated user.
+        This performs the same action as clicking on the Run Job button in the application.
+
+        :param recipe_id: o run a job, you just specify the recipe identifier (``wrangledDataset.id``).
+        If the job is successful, all defined outputs are generated,
+        as defined in the output object, publications, and writeSettings objects associated with the recipe.
+        To identify the ``wrangledDataset Id``, select the recipe icon in the UI flow view and
+        take the id shown in the URL, e.g. if the URL is /flows/10?recipe=7, the ``wrangledDataset`` Id is 7.
+        :type recipe_id: int.
+        """
+
+        recipe = {"wrangledDataset": {"id": recipe_id}}
```

Comment:
I had the impression that this payload can be much more complicated. At least that what I think looking at the documentation for example:
```
{
  "wrangledDataset": {"id": 1},
  "overrides": {
    "execution": "spark",
    "profiler": false,
    "writesettings": [
      {
        "path": "<path_to_output_file>",
        "action": "create",
        "format": "csv",
        "compression": "none",
        "header": false,
        "asSingleFile": false
      }
    ]
  }
}
```

Reply:
Yes, I am aware of that and will try to use this example and run DAG with this params. I am still confused when I trying ti use Dataprep.

Reply:
It's done and working.